contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/querydsl/mapping/CollectionFiltersImpl.java

```diff
@@ -1,15 +1,14 @@
 package com.contentgrid.spring.data.querydsl.mapping;
 
 import com.contentgrid.spring.querydsl.mapping.CollectionFilter;
-import java.util.List;
-import java.util.Map;
+import java.util.LinkedHashMap;
 import java.util.Optional;
 import java.util.stream.Stream;
 import lombok.RequiredArgsConstructor;
 
 @RequiredArgsConstructor
 class CollectionFiltersImpl extends AbstractCollectionFiltersImpl {
-    private final Map<String, CollectionFilter<?>> filters;
+    private final LinkedHashMap<String, CollectionFilter<?>> filters;
```

Comment:
Does this type have to be changed here?

Reply:
My reasoning was that if you want it to actually be ordered, then it should be changed in the class. But I guess that since this class is package-private, we're relying on nothing but the factory initializing it?